mlflow/protos/service.proto

```diff
@@ -346,6 +346,28 @@ service MlflowService {
     };
   }
 
+  // List all columns
+  //
+  rpc listAllColumns (ListAllColumns) returns (ListAllColumns.Response) {
+    option (rpc) = {
+      endpoints: [{
+        method: "POST",
+        path: "/mlflow/columns/list"
+        since { major: 2, minor: 0 },
+      }, {
+        method: "POST",
```

Comment:
You shouldn't use the POST method for this endpoint, IMO it should only be a GET